webshare/share-securecontext.http.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
@mikewest would know for sure, but I think that in order to run on a non-secure origin, you can name the test anything. I don't know if "http.html" means anything, but "https.html" does mean to run it over HTTPS, unsurprisingly.